src/commands/dev/dev.js

```diff
@@ -253,7 +254,29 @@ const dev = async (options, command) => {
     )
   }
 
-  await injectEnvVariables({ env: command.netlify.cachedConfig.env, site })
+  const startNetlifyGraphWatcher = Boolean(options.graph)
+  const netlifyToken = await command.authenticate()
+  let authlifyJWT
+
+  if (startNetlifyGraphWatcher) {
+    authlifyJWT = generateAuthlifyJWT(netlifyToken, siteInfo.authlify_token_id, site.id)
+  }
+
+  await injectEnvVariables({
+    env: Object.assign(
+      command.netlify.cachedConfig.env,
+      authlifyJWT == null
+        ? {}
+        : {
+            ONEGRAPH_AUTHLIFY_TOKEN: {
```

Comment:
Where do we want `ONEGRAPH_AUTHLIFY_TOKEN` injected to?

The PR description says builds and functions.
This injects it into functions and dev command, but not builds

Reply:
It's injected in builds via Bitballoon. 

Are there local builds where I should inject the variable too?

Reply:
The CLI has a build command https://github.com/netlify/cli/blob/e76355677415300eb3c69be17aeba76c177ec53d/src/commands/build/build.js#L24

The CLI gets all the configuration, including environment variables from `netlify/config`, so I wonder if it will be easier to implement this PR in that package.
See the code to retrieve site information in https://github.com/netlify/build/blob/dc76f4b27a9fe92395c7d20346ce1655f8340d8c/packages/config/src/api/site_info.js

Reply:
I feel like this is more specific to the CLI than the generic config package.

@erezrokah are you on board if we merge this with the current functionality to unblock Mathias's upcoming talk and make the build improvements later?